plugin/api.go

```diff
@@ -55,6 +55,11 @@ type API interface {
 	// DeleteUser deletes a user.
 	DeleteUser(userId string) *model.AppError
 
+	// GetUsers a list of users based on search options.
+	//
+	// Minimum server version: 5.8
```

Comment:
This will need to go into `5.10`. Could you please change this?

Reply:
```suggestion
	// Minimum server version: 5.10
```